package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "scripts": {
     "build": "turbo run build",
-    "dev": "turbo run dev",
+    "dev": "turbo run build && turbo run dev",
```

Comment:
So all the `dev` scripts still spin up watchers. However, a few of the packages kept erroring out when trying to start up because their dependencies weren't ready yet due to race conditions. For example, `packages/ui` might not have been kicked off and compiled soon enough before `web/sandbox` booted up.

If you look at `turbo.json`, we have the `build` command set up to compile in the proper order based on the dependency tree:

```
"build": {
      "dependsOn": ["^build"]
    },
```

I attempted to do something similar for `dev`:

```
"dev": {
      "dependsOn": ["^dev"]
    },
```

But the command just hangs and never resolves because the watch commands fired off by each `dev` command never exit with a return code.

So adding an initial `build` run before booting up `dev` is just a bandaid for now until we can dig deeper into the Turbo config.